changelog.md

```diff
@@ -25,6 +25,7 @@
                      section and safeguard the power system.
   * `RelayInfo.recloseFast`: True if recloseDelays are associated with a fast Curve, False otherwise.
   * `RegulatingControl.ratedCurrent`: The rated current of associated CT in amps for a RegulatingControl.
+  * `LoadShape`: added reactive power values to LoadShape.
```

Comment:
todo: This is a breaking change as you renamed `values` to `pValues` and changed the index of `normalise` which have been included in a previous build.

Reply:
Yep, it wasn't before but I changed that because I found values + qValues to be less explicit than pValues + qValues. I need to remember to not add reviewers to draft PRs. I forget that you get notified even if I say it is a draft.